packages/react/src/direction-provider/DirectionProvider.tsx

```diff
@@ -6,13 +6,7 @@ import { DirectionContext, type TextDirection } from './DirectionContext';
 /**
  * A provider that configures RTL/LTR behavior for part of an app or a whole app.
  *
- * Demos:
- *
- * - [Direction Provider](https://base-ui.com/components/react-direction-provider/)
- *
- * API:
- *
- * - [DirectionProvider API](https://base-ui.com/components/react-direction-provider/#api-reference-DirectionProvider)
+ * Documentation: [Base UI Direction Provider](https://base-ui.com/react/components/direction-provider)
```

Comment:
This should be under /react/utilites

Reply:
Nice catch, I managed to fumble `DirectionProvider` as `DirectoryProvider` somehow
https://github.com/mui/base-ui/pull/1015/commits/aad149816529a318271909182ffc02cf9e4e234d